docs/self_sign_cert.md

```diff
@@ -31,23 +22,88 @@ Default locale: en_NZ, platform encoding: UTF-8
 
-* Open `Preferences`
-* Go to `Authorithy Certificates`
-* Replace `CA Certificates KeyStore` with the `java runtime` from the previous step. Include `lib/security/cacerts` on the path.
+## 2. Load JRE CA Certificates
 
-#### 2. Import certificate
+Having identified your Java runtime location:
 
-* Hit `Open CA Certificates KeyStore` on the main window
-* On the tool bar, click on `Import Trusted Certificate`
+* Launch `KeyStore Explorer`
+* Select _Open an existing KeyStore_
+* Navigate to the Java runtime location identified previously, and then continue to open the file at `jre/lib/security/cacerts`.  If there is no `jre/` directory then this is a JRE distribution and should navigate and instead open the file at `lib/security/cacerts`.
+  * In the example above, this file would be `/Library/Java/JavaVirtualMachines/jdk1.8.0_172.jdk/Contents/Home/jre/lib/security/cacerts`.
+* You will likely be prompted for a password. The default password for the `cacerts` file is `changeit`.
+
+## 3. Import Self-Signed Certificate
+
+If you have the self-signed certificate in a file then:
+
+* Select _Tools > Import Trusted Certificate_
 * Select the certifcate file on disk
-* Give it a name, or use suggested name, hit `OK`
-* Hit ok on the success window
-* Last step, hit `Save`
+* Give it a name, or use suggested name, and click _OK_
+* Click _OK_ on the success window
+
+Otherwise use _Examine > Examine SSL_ to connect to your service and click the _Import_ button to import its SSL certificate. Then click _OK_.
+
+![Importing certificate with KeyStore Explorer](self_sign_cert-kse-import.png)
+
+## 4. Save the CA Certificates
+
+Now we save the updated keystore. We can either save to a new keystore and configure our build's JVM to use this new keystore as a _trusted keystore_, or modify the JRE's list of CA Certificates.
+
+#### Option 1: Create a New Trusted Keystore
+
+This option creates a _new_ list of CA Certificates and configures your build tool to use ths new list as the JRE's list of approved CA certificates, called the _trust store_.
+
+Within _KeyStore Explorer_, select _File > Save As..._ and save the new keystore file as a _JKS_ file alongside your build location. You will be prompted for a password; we use `password` in the examples below.
+
+##### Maven
+
+The following snippet shows how to configure Maven to use this new keystore file. You can configure the registry's username and password with [the `~/.m2/settings.xml` mechanism](https://github.com/GoogleContainerTools/jib/blob/master/jib-maven-plugin/README.md#using-maven-settings).
+
+```shell
+$ ./mvnw -Djavax.net.ssl.trustStore=path/to/keystore.jks \
+  -Djavax.net.ssl.trustStorePassword=password \
+  -Dimage=<host>:<port>/<image> jib:build
+```
+
+##### Gradle
+
+You may need to explicitly configure the auth [`username` and `password`](https://github.com/GoogleContainerTools/jib/tree/master/jib-gradle-plugin#using-specific-credentials).
```

Comment:
Ditto. This could be helpful, but I also think out of scope. I'm mostly concerned the user might be confused this a-bit-independent registry auth topic with this SSL cert setup.